pkg/scheduler/frameworkext/interface.go

```diff
@@ -132,6 +132,8 @@ type ReservationNominator interface {
 	AddNominatedReservation(pod *corev1.Pod, nodeName string, rInfo *ReservationInfo)
 	RemoveNominatedReservations(pod *corev1.Pod)
 	GetNominatedReservation(pod *corev1.Pod, nodeName string) *ReservationInfo
+	AddNominatedReservationToNode(reservePod *corev1.Pod, nodeName string)
```

Comment:
It's better to rename `AddNominatedReservationToNode` to `AddNominatedReservePod`.

Reply:
done